deploy/helm/kubecf/assets/operations/instance_groups/acceptance-tests.yaml

```
@@ -26,7 +26,7 @@
           credhub_secret: ((credhub_admin_client_secret))
           {{- end }}
           {{- if .Values.features.routing_api.enabled }}
-          include: '+tcp_routing'
+          include: '+tcp_routing,docker'
```

this shouldn't depend on tcp routing

```suggestion
          include: '+docker{{- if .Values.features.routing_api.enabled }}tcp_routing{{ - end }}'
```

---

an `{{else}}` line might be more readable

---

Oh, thanks for spotting that. I forgot that we put the include inside that conditional. Your suggestion only misses the comma. I'd keep your suggestion rather than adding a separate line for it.